sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/MetricDataMapper.java

```diff
@@ -143,9 +143,15 @@ public static void updateMetricPointBuilder(
                     pointBuilder.setCount((int) histogramCount);
                 }
                 HistogramPointData histogramPointData = (HistogramPointData) pointData;
+                double min = histogramPointData.getMin();
+                double max = histogramPointData.getMax();
+                if (shouldConvertToMilliseconds(metricData.getName(), isPreAggregatedStandardMetric)) {
+                    min = min * 1000;
+                    max = max * 1000;
+                }
                 pointDataValue = histogramPointData.getSum();
-                pointBuilder.setMin(histogramPointData.getMin());
-                pointBuilder.setMax(histogramPointData.getMax());
+                pointBuilder.setMin(min);
+                pointBuilder.setMax(max);
```

Comment:
can we do `sum` conversion here also (instead of below)?

Reply:
getSum is assigned to pointDataValue.. line 154 takes care of it.